src/xrpld/app/misc/detail/LendingHelpers.cpp

```diff
@@ -146,44 +166,41 @@ loanPrincipalFromPeriodicPayment(
     if (periodicRate == 0)
         return periodicPayment * paymentsRemaining;
 
-    /*
-     * This formula is the reverse of the one from the XLS-66 spec,
-     * section 3.2.4.1.1 (Regular Payment) used in loanPeriodicPayment
-     */
     return periodicPayment /
         computePaymentFactor(periodicRate, paymentsRemaining);
 }
 
+/* Splits gross interest into net interest (to vault) and management fee (to broker).
+ * Returns pair of (net interest, management fee).
+ *
+ * Equation (33) from XLS-66 spec, Section A-2 Equation Glossary
+ */
 std::pair<Number, Number>
 computeInterestAndFeeParts(
```

Comment:
There is an overload of this function on line 648. It'll be good if it's moved to right after this function.

Reply:
Fixed.